Formula/kahip.rb

```diff
@@ -20,20 +20,13 @@ class Kahip < Formula
   depends_on "open-mpi"
 
   on_macos do
-    depends_on "gcc"
+    depends_on "libomp"
   end
 
   def install
-    if OS.mac?
-      gcc_major_ver = Formula["gcc"].any_installed_version.major
-      ENV["CC"] = Formula["gcc"].opt_bin/"gcc-#{gcc_major_ver}"
-      ENV["CXX"] = Formula["gcc"].opt_bin/"g++-#{gcc_major_ver}"
-    end
```

Comment:
I guess `gcc` is part of dependency tree already due to `open-mpi` so could keep `gcc` as dependency and just modify to use Homebrew's compiler selection logic rather than manually overriding `CC` and `CXX`.

Reply:
Yes, that's the idea, gcc is already a dependency so we might as well use it rather than introduce another. I don't have anything against setting `CC` and `CXX`, but if you want to use some other mechanism, that would be acceptable as well.